torch/_torch_docs.py

```diff
@@ -6715,15 +6717,23 @@ def merge_dicts(*dicts):
         from the QR decomposition.
 
 Args:
-    input (Tensor): tensor of shape `(*, m, k)` where `*` is zero or more batch dimensions.
-    tau (Tensor): tensor of shape `(*, min(m, k))` where `*` is zero or more batch dimensions.
+    input (Tensor): tensor of shape `(*, mn, k)` where `*` is zero or more batch dimensions.
+    tau (Tensor): tensor of shape `(*, min(mn, k))` where `*` is zero or more batch dimensions.
     other (Tensor): tensor of shape `(*, m, n)` where `*` is zero or more batch dimensions.
-    left (bool): controls the order of multiplication
+    left (bool): controls the order of multiplication.
     transpose (bool): controls whether the matrix `Q` is conjugate transposed or not.
 
 Keyword args:
     out (Tensor, optional): the output Tensor. Ignored if `None`. Default: `None`.
 
+Raises:
+    RuntimeError:
+        if :attr:`input` and :attr:`other` have incompatible shapes for the matrix multiplication.
+
+    RuntimeError:
+        if size of :attr:`tau` vector is larger than number of rows (when ``left=True``)
```

Comment:
nit. "if the size of :attr:`tau` is larger than the number", or even better, "the length".
Also, without double quotes, it should be `` `:attr:`left`\ `= True` `` (same in the next line).